### PR TITLE
Expo build issue

### PR DESCRIPTION
**Pull request by chandantiwari1**
made some changes in the repo to avoid eas  build errors

### CODE REVIEW

**Comment by sutyum** [comment]
Issue with yarn workspace resolved? If so, describe how and merge conflicts to proceed to merge @chandantiwari1

**Comment by chandantiwari1** [comment]
> Issue with yarn workspace resolved? If so, describe how and merge conflicts to proceed to merge @chandantiwari1

yes issue with the yarn workspace has been. I delete  app.json, eas.json, tsconfig.json and these files are no longer needed. files which were in the root directory of our app 
reference: [pnpm monorepo](https://github.com/byCedric/expo-monorepo-example)
chan

File: apps/meway-tablet/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@meway/meway-tablet",
   "version": "0.0.0",
-  "main": "index.ts",
+  "main": "index.js",
```
Comment (sutyum) [inline apps/meway-tablet/package.json:5]: why

File: apps/meway-tablet/src/constants/Colors.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-anonymous-default-export */
+const tintColor = "#4630ec";
```
Comment (sutyum) [inline apps/meway-tablet/src/constants/Colors.ts:2]: use tailwind

File: apps/meway-tablet/src/screens/FaceCount.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          runClassifications: FaceDetector.FaceDetectorClassifications.none,
+          minDetectionInterval: 100,
+          tracking: true,
+        }}
+      />
+    </>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (sutyum) [inline apps/meway-tablet/src/screens/FaceCount.tsx:72]: use tailwind/nativewind

File: apps/meway-tablet/tsconfig.json [hunk 4]
```diff
@@ -5,5 +5,5 @@
     "jsx": "react-native",
 
   },
-  "include": ["./src/**/*","index.ts"],
+  "include": ["./src/**/*","index.js"],
```
Comment (sutyum) [inline apps/meway-tablet/tsconfig.json:8]: why

File: package.json [hunk 5]
```diff
@@ -1,32 +1,38 @@
 {
-    "name": "meway",
-    "version": "0.0.1",
-    "private": true,
-    "workspaces": [
```
Comment (sutyum) [inline package.json:5]: How will packages be linked without this?

**Review by sutyum — CHANGES_REQUESTED** [review]

File: apps/meway-tablet/package.json (hunk 1 above)
Comment (chandantiwari1) [reply apps/meway-tablet/package.json:5]: sir we can use any of then either ts or js . it doesn't affect the build process.

File: apps/meway-tablet/src/constants/Colors.ts (hunk 2 above)
Comment (chandantiwari1) [reply apps/meway-tablet/src/constants/Colors.ts:2]: ok sir

File: apps/meway-tablet/src/screens/FaceCount.tsx (hunk 3 above)
Comment (chandantiwari1) [reply apps/meway-tablet/src/screens/FaceCount.tsx:72]: ok sir

File: apps/meway-tablet/tsconfig.json (hunk 4 above)
Comment (chandantiwari1) [reply apps/meway-tablet/tsconfig.json:8]: sir we can use any of then either ts or js . it doesn't affect the build process.

File: package.json (hunk 5 above)
Comment (chandantiwari1) [reply package.json:5]: sir i arranged them same as turbo repo docs, i didn't remove them.

**Review by sutyum — APPROVED** [review]

**Pull request merged**